gtsfm/scene_optimizer.py

```diff
@@ -168,8 +159,22 @@ def create_computation_graph(
             i2Ui1_graph_dict,
             v_corr_idxs_graph_dict,
             camera_intrinsics_graph,
+            gt_pose_graph
         )
 
+        # aggregate metrics
+        if gt_pose_graph is not None:
```

Comment:
i would prefer to keep `aggregate_frontend_metrics` before the multi-view step, so front-end metrics get dumped by Dask, even if the back-end crashes.